panacea/grpc_client.go

```diff
@@ -123,6 +123,20 @@ func (c *GrpcClient) GetChainId() (string, error) {
 	return response.GetBlock().GetHeader().ChainID, nil
 }
 
+func (c *GrpcClient) GetRegisteredDataValidator(address string) (datapooltypes.DataValidator, error) {
+	client := datapooltypes.NewQueryClient(c.conn)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	res, err := client.DataValidator(ctx, &datapooltypes.QueryDataValidatorRequest{Address: address})
```

Comment:
This is something I haven't been able to review before.
If the `DataValidator` is not reigstered, it does not respond with an error and returns an empty value.
How about responding with an error if `DataValidator` is not registered?